wicket-tester/src/main/java/module-info.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module org.apache.wicket.tester {}
```

Comment:
it will need to export the package with the moved classes

Reply:
wicket-tester was in this PR only a test and at no place used. Sorry for the confusion, I shouldn't have that commited with here (in https://github.com/apache/wicket/pull/659)... In https://github.com/apache/wicket/pull/660, _exports org.apache.wicket.util.tester;_ does already exist.